### PR TITLE
chore: Write README for all packages except tokens

### PR DESCRIPTION
**Pull request by VincentSmedinga**
# Describe the pull request

## What

This is a continuation of #1655 to improve the general documentation of all our packages.

It also removes mentions of Salesforce, as we no longer aim to replicate our components for it. I’ve removed more ‘future plans’ and ‘roadmaps’ as they easily get outdated.

## Why

To help people understand what we offer and how they can use it.

## How

Adds or updates the README and some fields in package.json for the assets, css, react, and react-icons packages.

The sections headings are consistently as follows, with all of them optional:

1. About
2. Motivation
3. Installation
4. Usage
5. Updating
6. Support

## Checklist

Before submitting your pull request, please ensure you have done the following. Check each checkmark if you have done so or if it wasn't necessary:

- [x] Add or update unit tests
- [x] Add or update documentation
- [x] Add or update stories
- [x] Add or update exports in index.\* files
- [x] Start the PR title with a Conventional Commit prefix, [as explained here](https://github.com/Amsterdam/design-system/blob/main/documentation/publishing.md?plain=1#L11).

## Additional notes

- Provide links to any related issues or discussions.
- Add a link to the specific story in the feature branch deploy.
- Mention any areas where additional review or feedback is needed.

### CODE REVIEW

File: storybook/package.json [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
   "license": "EUPL-1.2",
   "name": "@amsterdam/storybook",
   "keywords": [
-    "nl-design-system"
+    "amsterdam",
+    "amsterdam-design-system",
+    "design-system",
+    "nl-design-system",
+    "storybook"
   ],
```
Comment (alimpens) [inline storybook/package.json:14]: I'm not sure how relevant this is, we don't publish this package

File: packages/css/README.md [hunk 2]
```diff
@@ -1,27 +1,87 @@
...
+
+## Motivation
+
+Publishing the stylesheets as a separate package saves a lot of work when replicating components in other libraries, platforms, or Saas-applications.
+This is a powerful approach to implement the design system across different technologies.
+
+The classes act like a contract between the component’s definition and its implementations.
+As a bonus, updates to tokens or styles can be propagated to all of them.
+
+## Installation
```
[the remark this answers is not in the document]
Comment (VincentSmedinga) [reply packages/css/README.md:16]: Fair point, but I’d also like people to understand what they install before rushing in.

I’ve renamed ‘About‘ to ‘Introduction’, which may even help people skip it, moved some text to ‘Usage’ and made the rest more compact.

File: packages/css/README.md [hunk 3]
```diff
@@ -1,27 +1,87 @@
...
+
+The stylesheets reference our [design tokens](https://www.npmjs.com/package/@amsterdam/design-system-tokens).
+Both packages need to be installed.
+
+```sh
```
[the remark this answers is not in the document]
Comment (VincentSmedinga) [reply packages/css/README.md:21]: It should work everywhere, it did ‘work on my machine’, and looks better – but maybe better safe than sorry.

I’ve removed the `pnpm` and `yarn` commands in the same spirit and made the language descriptor in the backticks consistent.

File: packages/css/README.md [hunk 4]
```diff
@@ -1,27 +1,87 @@
...
+```
+
+## Usage
+
+### In JavaScript
+
+Import the main stylesheet and use the class names in your markup.
+
+```ts
+import "@amsterdam/design-system-tokens/dist/index.css";
```
[the remark this answers is not in the document]
Comment (VincentSmedinga) [reply packages/css/README.md:40]: Added it, and the font CSS as well.

File: packages/css/README.md [hunk 5]
```diff
@@ -1,27 +1,87 @@
...
+    <link rel="stylesheet" href="node_modules/@amsterdam/design-system-css/dist/index.css" />
   </head>
-  <body>
+  <body class="ams-body">
     <p class="ams-paragraph">Hello, world!</p>
   </body>
 </html>
 ```
 
+### With CSS Modules
```
Comment (VincentSmedinga) [inline packages/css/README.md:66]: Forgot to ask earlier: I lack experience with CSS modules, so I left the JS snippet below untouched. Can you check it @alimpens?

- Does it need the class component syntax?
- And the empty fragment?
- And the explicit return statement?
- How should we add the font and token imports?

File: packages/react/README.md [hunk 6]
```diff
@@ -1,18 +1,92 @@
...
+This package is the primary entry point for digital services that we build in-house.
+We’ve adopted the architecture of [NL Design System](https://nldesignsystem.nl/).
 
-The React Library has not yet reached a 1.0.0 version.
-However, most of them are stable enough to be used in production.
-Components that have known issues, or for which we anticipate API changes, show a ‘beta’ badge on their page.
+## Installation
 
-Make sure you specify the exact version as dependency.
-You can then schedule an upgrade to the latest version when you have time to test for regressions.
```
[the remark this answers is not in the document]
Comment (VincentSmedinga) [reply packages/react/README.md:18]: I’ve restored and shortened it into the ‘Stability’ section below.

File: proprietary/react-icons/README.md [hunk 7]
```diff
@@ -1,5 +1,48 @@
 <!-- @license CC0-1.0 -->
 
-# Icons React
+# Amsterdam Design System: React icons
 
-Icon svgs as React components
+This package provides all icons as React components from the [Amsterdam Design System](https://designsystem.amsterdam).
```
[the remark this answers is not in the document]
Comment (VincentSmedinga) [reply proprietary/react-icons/README.md]: Updated; in the `package.json` description as well.

File: storybook/package.json (hunk 1 above)
Comment (VincentSmedinga) [reply storybook/package.json:14]: The `keywords` property is required. Do you prefer an empty array over consistency with the other packages?

File: packages/css/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (alimpens) [reply packages/css/README.md:21]: > Newline characters should be properly ignored in this case

They aren't on Windows machines / a WSL terminal, which is what most devs working for the City use.

File: packages/css/README.md (hunk 5 above)
Comment (alimpens) [reply packages/css/README.md:67]: Hm, I missed this. I'm not sure we should keep this section at all. In Next and Vite the CSS file has to end in `*.module.css` to (automaticaly) be recognized as a CSS module, which means this example doesn't work with ADS and Vite / Next.

File: proprietary/react-icons/README.md [hunk 8]
```diff
@@ -1,5 +1,40 @@
...
+
+## Usage
+
+Import the component for the icon you need and use it in your JSX.
+
+```ts
+import { SearchIcon } from '@amsterdam/design-system-react-icons'
+
+export const App = () => (
+  <div>
```
Comment (alimpens) [inline proprietary/react-icons/README.md:26]: Seeing as we're already in a React project, maybe we should use Icon here as well?

File: storybook/package.json (hunk 1 above)
Comment (alimpens) [reply storybook/package.json:14]: > The keywords property is required.

No it isn't?

File: packages/react/README.md [hunk 9]
```diff
@@ -1,18 +1,71 @@
...
+Then import and use the components in your JSX.
+
+```ts
+import "@amsterdam/design-system-assets/font/index.css";
+import "@amsterdam/design-system-css/dist/index.css";
+import "@amsterdam/design-system-tokens/dist/index.css";
+import { Paragraph, Screen } from "@amsterdam/design-system-react";
+
+const App = () => (
+  <Screen>
```
Comment (alimpens) [inline packages/react/README.md:41]: Do we have a good reason to use Screen here? I prefer these examples to be as minimal as possible without looking broken.

File: packages/css/README.md (hunk 5 above)
Comment (VincentSmedinga) [reply packages/css/README.md]: Ah! Removed it.

File: storybook/package.json (hunk 1 above)
Comment (VincentSmedinga) [reply storybook/package.json:14]: [It is for us :)](https://github.com/Amsterdam/design-system/blob/1599936e7f647004a449083b8bfe2071d422f983/.npmpackagejsonlintrc.json#L29)

Comment (alimpens) [reply storybook/package.json:14]: Ah, wasn't aware of that. I don't really see why it should be required though, we can remove that rule imo. Or an empty array is fine as well, if we want to keep the rule.

Comment (VincentSmedinga) [reply storybook/package.json]: I’ve emptied it.

**Review by alimpens — APPROVED** [review]

**Pull request merged**